articles/network-watcher/network-watcher-packet-capture-manage-cli.md

```diff
@@ -100,29 +106,21 @@ Once the preceding steps are complete, the packet capture agent is installed on
 
 ### Step 1
 
-The next step is to retrieve the Network Watcher instance. TThe name of the Network Watcher is passed to the `az network watcher show` cmdlet in step 4.
-
-```azurecli
-az network watcher show --resource-group resourceGroup --name networkWatcherName
-```
-
-### Step 2
-
 Retrieve a storage account. This storage account is used to store the packet capture file.
 
 ```azurecli
-azure storage account list
+az storage account list
 ```
 
-### Step 3
+### Step 2
 
```

Comment:
This doc could be written better. Let's take the opportunity to fix some low-hanging fruit.

When describing a formal step in a process, the first line of the description should usually be imperative, i.e. stating what to do next. Instead, this one opens up by talking about filters without establishing the context.

Let's add a sentence here to the effect of: "At this point you are ready to create a packet capture. First, let's examine the parameters that you may want to configure."